app/lib/theme/DangerAlert.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Alert } from "@button-inc/bcgov-theme";
+
+const DangerAlert: React.FC<{ proceedOnClick; cancelOnClick }> = ({
```

Comment:
we should use a real interface and types for these methods. We can also give them better names:

```tsx
interface Props {
  onProceed: () => void;
  onCancel: () => void;
}

const DangerAlert: React.FC<Props> = ...
```